### PR TITLE
scheduler: add better hot region output for pd-ctl

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

image
image

It's a part of #3201

### Check List

Tests

- Unit test

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: server/statistics/hot_peer.go [hunk 1]
```diff
@@ -27,27 +27,27 @@ const (
 
 // HotPeerStat records each hot peer's statistics
 type HotPeerStat struct {
-	StoreID  uint64 `json:"store_id"`
-	RegionID uint64 `json:"region_id"`
+	StoreID  uint64 `json:"store-id"`
```
Comment (rleungx) [inline server/statistics/hot_peer.go:30]: Seem store id is not necessary to output

**Comment by rleungx** [comment]
BTW, will it affect TiDB?

File: server/statistics/hot_peer.go (hunk 1 above)
Comment (lhy1024) [reply server/statistics/hot_peer.go:30]: yep

**Comment by lhy1024** [comment]
> BTW, will it affect TiDB?

I am not sure. How will we check it?

**Comment by lhy1024** [comment]
@rleungx @HunDunDM @disksing PTAL

File: server/schedulers/utils.go [hunk 2]
```diff
@@ -333,8 +333,8 @@ func (li *storeLoadDetail) toHotPeersStat() *statistics.HotPeersStat {
 		peers = append(peers, *peer.Clone())
 	}
 	return &statistics.HotPeersStat{
-		TotalBytesRate: li.LoadPred.Current.ByteRate,
-		TotalKeysRate:  li.LoadPred.Current.KeyRate,
+		TotalBytesRate: float64(int(li.LoadPred.Current.ByteRate)),
```
Comment (HunDunDM) [inline server/schedulers/utils.go]:
```suggestion
		TotalBytesRate: math.Round(li.LoadPred.Current.ByteRate),
```

File: server/statistics/hot_peer.go [hunk 3]
```diff
@@ -88,26 +86,26 @@ func (stat *HotPeerStat) IsNew() bool {
 
 // GetByteRate returns denoised BytesRate if possible.
 func (stat *HotPeerStat) GetByteRate() float64 {
-	if stat.RollingByteRate == nil {
-		return stat.ByteRate
+	if stat.rollingByteRate == nil {
+		return float64(int(stat.ByteRate))
```
Comment (HunDunDM) [inline server/statistics/hot_peer.go]: ditto

File: server/statistics/hot_peer.go [hunk 4]
```diff
@@ -88,26 +86,26 @@ func (stat *HotPeerStat) IsNew() bool {
...
 	}
-	return stat.RollingByteRate.Get()
+	return float64(int(stat.rollingByteRate.Get()))
```
Comment (HunDunDM) [inline server/statistics/hot_peer.go]: ditto

File: server/statistics/hot_peer.go [hunk 5]
```diff
@@ -88,26 +86,26 @@ func (stat *HotPeerStat) IsNew() bool {
...
 
 // GetKeyRate returns denoised KeysRate if possible.
 func (stat *HotPeerStat) GetKeyRate() float64 {
-	if stat.RollingKeyRate == nil {
-		return stat.KeyRate
+	if stat.rollingKeyRate == nil {
+		return float64(int(stat.KeyRate))
 	}
-	return stat.RollingKeyRate.Get()
+	return float64(int(stat.rollingKeyRate.Get()))
```
Comment (HunDunDM) [inline server/statistics/hot_peer.go]: ditto

File: server/statistics/hot_peer.go [hunk 6]
```diff
@@ -88,26 +86,26 @@ func (stat *HotPeerStat) IsNew() bool {
...
-	return stat.RollingByteRate.Get()
+	return float64(int(stat.rollingByteRate.Get()))
 }
 
 // GetKeyRate returns denoised KeysRate if possible.
 func (stat *HotPeerStat) GetKeyRate() float64 {
-	if stat.RollingKeyRate == nil {
-		return stat.KeyRate
+	if stat.rollingKeyRate == nil {
+		return float64(int(stat.KeyRate))
```
Comment (HunDunDM) [inline server/statistics/hot_peer.go]: ditto

**Review by HunDunDM — COMMENTED** [review]
rest LGTM

**Review by HunDunDM — APPROVED** [review]
LGTM

**Pull request merged**